src/app/lib/analyticsUtils/analytics.const.js

```diff
@@ -1,6 +1,6 @@
 const SUPPORTED_MEDIUM_CAMPAIGN_TYPES = [
   'affiliate',
-  's1',
+  'sl',
```

Comment:
change made due to [value from doc](https://developers.atinternet-solutions.com/javascript-en/campaigns-javascript-en/marketing-campaigns-v2/#sponsored-links_6)

Reply:
nopes, it was stated as `sl` in the original issue